tests/auth/answer-post.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, test } from "@playwright/test";
+
+const randomNumber = Math.floor(Math.random() * 1000).toString();
+const replyContent = `This is a test reply from football fanatics ${randomNumber}`;
+
+test("should create and delete a reply to a post", async ({ page }) => {
+  await page.goto("/");
+  await page.getByRole('button', { name: 'Comment' }).first().click();
+
+  await expect(page).toHaveURL(/\/posts\/[A-Z0-9]+/);
+  await expect(page.getByText('Picture Upload')).toBeVisible();
+
+  await page.getByLabel('Send').isDisabled();
+  await page.getByPlaceholder('What is your opinion about').fill(replyContent);
```

Comment:
I think we could use here a labels dictionary. A simple object containing the labels used in the tests and in the components (something like constants). This way, when a label gets updated the tests would get these automatically. 
We would need to check in all tests and detect which labels are being used and if they are being set in the NextApp and not in the designLibrary. What do you think about this?

Reply:
good idea. this would make the tests more robust

Reply:
@lehmi11, this looks pretty good. You added more robustness to the app. Thanks